### ISSUE TITLE
[CI] openblas build failed in static build

### ISSUE THREAD

**Issue by TaoLv** [apache/incubator-mxnet#16021]
I notice the openblas 0.3.5 build failed in static build but the CI still passed.

```
../libopenblasp-r0.3.5.a(cblas_sgemm.o): In function `cblas_sgemm':
gemm.c:(.text+0x57f): undefined reference to `sgemm_kernel_direct_performant'
gemm.c:(.text+0x5d8): undefined reference to `sgemm_kernel_direct'
collect2: error: ld returned 1 exit status
make[1]: *** [xscblat3] Error 1
make[1]: *** Waiting for unfinished jobs....
make: *** [tests] Error 2
++++ make --quiet -j 72 PREFIX=/work/mxnet/staticdeps install
make[1]: warning: -jN forced in submake: disabling jobserver mode.
Generating openblas_config.h in /work/mxnet/staticdeps/include
Generating f77blas.h in /work/mxnet/staticdeps/include
Generating cblas.h in /work/mxnet/staticdeps/include
Copying LAPACKE header files to /work/mxnet/staticdeps/include
Copying the static library to /work/mxnet/staticdeps/lib
Copying the shared library to /work/mxnet/staticdeps/lib
install: cannot stat 'libopenblasp-r0.3.5.so': No such file or directory
make[1]: *** [install] Error 1
make: *** [install] Error 2
```
For example:
http://jenkins.mxnet-ci.amazon-ml.com/blue/organizations/jenkins/mxnet-validation%2Funix-cpu/detail/PR-15983/3/pipeline/297

**Comment by TaoLv** [comment]
@zachgk @lanking520 @szha

**Comment by TaoLv** [comment]
Ping @zachgk  @lanking520. Could please take a look at this issue?

**Comment by lanking520** [comment]
Let me try it locally...

**Comment by perdasilva** [comment]
Bumping openblas to v0.3.7 seems to fix the issue =)

**Comment by TaoLv** [comment]
Thank you @perdasilva. Do you have a fix for that? The mkldnn-v1.0 feature branch is also suffering from this issue.
http://jenkins.mxnet-ci.amazon-ml.com/blue/organizations/jenkins/mxnet-validation%2Funix-cpu/detail/PR-16012/3/pipeline

**Comment by perdasilva** [comment]
@TaoLv I've just opened a PR, please review and merge if ok =D
